ui/StatusQ/sandbox/main.qml

```diff
@@ -146,11 +146,14 @@ StatusWindow {
             }
 
             leftPanel: StatusScrollView {
+                id: sectionsScrollView
                 anchors.fill: parent
-                anchors.topMargin: 48
+                topPadding: 48
+                contentWidth: availableWidth
```

Comment:
That would be a wrong thing to do. It's user's decision to set the `contentWidth`. 
Yes, in like 80% cases of using `StatusScrollView` we will set `contentWidth: availableWidth`. Because we have mostly vertical scroll views. But I think we should leave this to end user. Just as original Qt's `ScrollView` doesn't set it by default.